### DISCUSSION TITLE
Homepage links unclickable on iOS after next build

### DISCUSSION THREAD

**Discussion by ImJustChew** [vercel/next.js#63571] [Help]
### Summary

I have a very interesting issue rn, and I am not able to debug the cause.
tldr; on my homepage on iOS, links have to be pressed multiple times for it to register, and it only happens on the homepage. 
Once you switch to another page, the issue disappears. 
I tried accessing the homepage via dev server on my iOS device, **THERE ARE NO ISSUES FOUND.** all presses are registering on the development version. 

Now apart from security keys, prod and dev should be identical, the page is hosted on vercel, and that is where I can no longer find hints. I would really value some knowledge or help from anyone. 

Update: I've tried to disable both next-pwa and sentry in next.config, doesn't make a difference.

Access prod [here](https://nthumods.com)
and the latest dev build here [here](https://courseweb-git-merging-course-nthumods.vercel.app/)

Repo is [here](https://github.com/nthumodifications/courseweb) too. (Btw pls don't judge my bad code)

Thanks ❤️

**Comment by Joshimello** [comment]
Tested on IOS, not responsive as expected

**Comment by mshkfrd** [comment]
same issue
